identity/manager.go

```diff
@@ -88,6 +88,25 @@ func (m *Manager) Create(ctx context.Context, i *Identity, opts ...ManagerOption
 	return m.r.PrivilegedIdentityPool().CreateIdentity(ctx, i)
 }
 
+func (m *Manager) CreateIdentities(ctx context.Context, identities []*Identity, opts ...ManagerOption) (err error) {
```

Comment:
Tests please. For example, are identities properly validated? Is the Identity JSON Schema properly being applied?

Reply:
This is now tested through handler and persister tests.